WcaOnRails/app/views/registrations_mailer/notify_registrant_of_locked_account_creation.html.erb

```diff
@@ -0,0 +1,10 @@
+<p>
+  Your registration for <%= @competition.name %> has been imported to the WCA website
+  and as a result a locked account has been created for you.
```

Comment:
1. I don't think so. Normally when sending an email to registrant, we would check `@registration.user.locale`, but for a newly created locked account we don't have that info. We could take a guess based on their country, but I thought it's alright to keep it in English. Any preference here?
2. Agreed, I will update the wording.
    - concerning side note: initially I planned to just include an activation URL, but soon I realized the tokens have expiration time (and we definitely don't want an exception in this case). So I figured reusing the existing flow is just fine, and more secure than alternatives. Honestly I think most users don't keep their emails open, so in most cases we didn't *just* email them ^^
3. Yeah, I thought of an alternative where you get a clear message during account creation instead. It has one caveat though, if someone (let's assume with WCA ID) registered for a competition using one email, and signs up with another, then not only does he unnecessarily create a duplicated account, but also he cannot claim his WCA ID from the new account, as it's already claimed to the locked account. I don't have strong preference on that, email just seems fine. We can implement both if you like, i.e. leave email notifications, but also improve the error message if someone signs up with an email belonging to a locked account. What do you prefer?

Reply:
I've updated the wording like so:
![image](https://user-images.githubusercontent.com/17034772/50531555-3e0c0900-0b0c-11e9-94e9-4a7a1d8030a1.png)
How does it sound?

Reply:
> but for a newly created locked account we don't have that info

Oh, right, duh! This seems fine to me =)

> registered for a competition using one email, and signs up with another, then not only does he unnecessarily create a duplicated account, but also he cannot claim his WCA ID from the new account, as it's already claimed to the locked account

Ah, good point! I *think* that if they try to claim their WCA ID while creating the account, then they will get a message about that WCA ID already being assigned to a different account, but it would not be easy for them to figure out which account that is.

I don't know that this email will actually prevent that situation. We already get people who don't really understand email and accounts and passwords creating lots of duplicate accounts. =( However, I'm sold on keeping the email! I also like the idea of improving the signup flow anyways, but we can take care of that later.

> I've updated the wording like so:

I like it! I would like to add a sentence or two about the purpose of these accounts, though. Maybe something like this?

> With this account, you can add a picture to your WCA results profile page, and register for other competitions around the world.

Feel free to reword. I just feel pretty bad about emailing someone about something without telling them *why* they should care about it.